.travis.yml

```diff
@@ -152,6 +152,7 @@ matrix:
             - travis_retry hide-logs.sh ant $OPTS -f platform/masterfs test
             - hide-logs.sh ant $OPTS -f platform/masterfs.linux test
             - hide-logs.sh ant $OPTS -f platform/netbinox test  -Dtest.config=stableBTD
+            - hide-logs.sh ant $OPTS -f platform/o.n.agent test
```

Comment:
this job moved to github. This could be added to `platform-modules-test1` or `platform-modules-test2`
https://github.com/apache/netbeans/blob/master/.github/workflows/main.yml#L727

test1 is only JDK 8. test2 has some JDK 11 tests too.